packages/stencil-component/test-data/ej-fra/index-missing-alignment.html

```diff
@@ -15,5 +15,5 @@
     </body>
 
     <!-- The last step needed is to import the package -->
-    <script type="module" src='//localhost:3333/build/read-along.esm.js'></script>
+    <script type="module" src='//0.0.0.0:3333/build/read-along.esm.js'></script>
```

Comment:
does this work?!?

Reply:
But I've already set it to 127... on my rebased copy of this branch.